### DISCUSSION TITLE
Add example of using age with minisign/signify

### DISCUSSION THREAD

**Discussion by magical** [FiloSottile/age#253] [General]
There have been several requests for signing support. Perhaps an example in the readme showing how to use age in conjunction with a third-party signing tool like minisign would help?

**Comment by tunnckoCore** [comment]
@magical yea... i'm trying to understand that idea. I tried. They both of them has other keys format, both sides are not compatible with each other. You can't just sign using age private key, like `signify -S -s age-keys/alice.sec -m my-file`, it errors.

cc @FiloSottile
